src/commands/install.ts

```diff
@@ -33,6 +35,14 @@ export interface InstallOptions {
    * allow installation on CI environments
    */
   installOnCI?: boolean;
+  /**
+   * Enable monorepo mode
+   */
+  monorepo?: boolean;
```

Comment:
A. This is expected to be put in the root `aicm.json`? 
B. I wonder regarding the term `monorepo` vs. `workspaces` term, have you considered it?

Reply:
Yeah, I actually debated whether to use `workspaces` or `monorepo` here. I'm not sure what's more well known.

A good reason to go with workspaces is that we could also generate a [vscode multi-root workspaces](https://code.visualstudio.com/docs/editing/workspaces/workspaces) configuration file based on the places you've placed the `aicm.json` config. This is also the term used in yarn.

What do you think? should we change this to workspaces?